pkg/models/reward.go

```diff
@@ -0,0 +1,7 @@
+package models
+
+type RanklistResponse struct {
+	NickName    string `json:"nick_name"`
+	Rewards     int    `json:"reward"`
```

Comment:
Use consistent names.
```suggestion
	Rewards     int    `json:"rewards"`
```

Reply:
2 different names issue:

1. In API docs /comment/course_group/<id> GET (edited during the meeting), "reward" is used in response
2. In #4 , field name "rewards"

Reply:
Let's just change all these occurences to `reward`. (Mostly because we want a stable frontend interface and it seems to make more sense to use singular form here)